src/NuGetGallery.Core/GitHub/NuGetPackageGitHubInformation.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+
+namespace NuGetGallery.GitHub
+{
+    public class NuGetPackageGitHubInformation
+    {
+        public int TotalRepos { get; set; }
```

Comment:
C# style is for properties to be listed after the constructor (properties are really just syntactic sugar for methods)